cmake/TaichiCAPITests.cmake

```diff
@@ -17,12 +17,25 @@ add_executable(${C_API_TESTS_NAME} ${TAICHI_C_API_TESTS_SOURCE})
 
 target_link_libraries(${C_API_TESTS_NAME} PRIVATE taichi_c_api)
 target_link_libraries(${C_API_TESTS_NAME} PRIVATE gtest_main)
+target_link_libraries(${C_API_TESTS_NAME} PRIVATE ${llvm_libs})
 
 target_include_directories(${C_API_TESTS_NAME}
   PRIVATE
     ${PROJECT_SOURCE_DIR}
     ${PROJECT_SOURCE_DIR}/c_api/include
     ${PROJECT_SOURCE_DIR}/c_api/src
+    ${PROJECT_SOURCE_DIR}/c_api/tests
```

Comment:
OOC, with C-API, why do we still need these headers?

Reply:
hmmm, this is an interesting question. 

To make a long story short, right now GUI is not compatible with C-APi Vulkan backend, therefore I need to use "VulkanDevice" to workaround this issue when writing the Mpm88 Demo: https://github.com/taichi-dev/taichi-aot-demo/tree/master/mpm88_desktop.

The build system for CAPI tests will get better once GUI get merged into C-API

Reply:
okay I give up, let's not keep the work around in C-API test